packages/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -180,48 +165,6 @@ function createCompiler({
       errors: true,
     });
 
-    if (useTypeScript && statsData.errors.length === 0) {
```

Comment:
Why is this being removed? Is it no longer needed with webpack 5?

Reply:
As I remember the new fork-ts-checker plugin will output issues (so removing the output in CRA removed the double output)

Not sure if @piotr-oles can confirm that my memory is not failing me? :)

Reply:
Yes, the plugin supports webpack dev server OOTB 
 * https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/main/src/hooks/interceptDoneToGetWebpackDevServerTap.ts
  * https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/blob/main/src/hooks/tapDoneToAsyncGetIssues.ts#L71

Reply:
thanks @piotr-cz!
@iansu I'll resolve comment, the code removed had to do with outputting type issues - these are now outputted directly from fork-ts-checker to webpack instead (should be similar in behavior)